src/components/Sketches/components/EditSketchModal.js

```diff
@@ -264,30 +264,26 @@ class EditSketchModal extends React.Component {
     );
   }
 
-  renderThumbnailModal() {
-    const icons = ThumbnailArray.map((val, index) => {
-      return (
-        <figure
-          className="sketches-gallery-item"
-          key={val}
-          onClick={() => this.setState({ newThumbnail: index })}
-        >
-          <img
-            src={`${process.env.PUBLIC_URL}/img/sketch-thumbnails/${val}.svg`}
-            className={
-              "sketches-gallery-img" + (this.state.newThumbnail === index ? "-selected" : "")
-            }
-            alt="icon"
-          />
-        </figure>
-      );
-    });
+  const renderThumbnailModal = () => {
+    const icons = ThumbnailArray.map((val, index) => (
+      <figure
+        className="sketches-gallery-item"
+        key={val}
+        onClick={() => setNewThumbnail(index)}
```

Comment:
`figure` is not an interactive element by default, so you could add `role="button"` to mark it as interactive

Reply:
Used `role="presentation"` instead, for some reason "button" didn’t work